tasks/examples/client-tasks.rs

```diff
@@ -16,25 +16,12 @@ async fn main() {
     let client = ClientFlow::new(ClientFlowOptions::default());
     let mut scheduler = Scheduler::new(client);
 
-    let handle1 = scheduler.enqueue_task(CapabilityTask::default());
-
-    loop {
-        match stream.progress(&mut scheduler).await.unwrap() {
-            SchedulerEvent::TaskFinished(mut token) => {
-                if let Some(capability) = handle1.resolve(&mut token) {
-                    println!("handle1: {capability:?}");
-                    break;
-                }
-            }
-            SchedulerEvent::Unsolicited(unsolicited) => {
-                println!("unsolicited: {unsolicited:?}");
-
-                if let Response::Status(Status::Bye { .. }) = unsolicited {
-                    break;
-                }
-            }
-        }
-    }
+    let capabilities = stream
+        .progress(scheduler.run_task(CapabilityTask::new()))
+        .await
+        .unwrap()
+        .unwrap();
+    println!("capabilities: {capabilities:#?}");
 
     let handle2 = scheduler.enqueue_task(AuthenticateTask::plain("alice", "pa²²w0rd", true));
     let handle3 = scheduler.enqueue_task(LogoutTask::default());
```

Comment:
`handle1` is missing now. Let's rename the variables to `authenticate_handle` and `logout_handle`.